spec/hydra/works/performance_tests/repeat_gets_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'spec_helper'
+
+describe 'repeat_gets' do
```

Comment:
instead of `xit` on all the tests, how about a condition in the describe like: `if: ENV['PERFORMANCE_TESTS']`